projects/EIP-x_stateless_verification_with_LC_which_consume_ZKP_Portal.md

```diff
@@ -0,0 +1,71 @@
+
+##EIP-x 
+ Stateless LC That Consumes ZKP With Efficient Access To The Specific Segments Of The State.
+
+
+## Motivation
+In the dynamic landscape of blockchain technology, addressing the limitations of traditional light clients is paramount for the widespread adoption and efficient functioning of the Ethereum network. Conventionally, light clients have burdened full nodes by incrementally requesting block state, posing challenges to network scalability. The integration of Verkle trees has provided a significant step forward, facilitating smoother transitions for lightweight clients between blocks. However, a lingering concern lies in the assurance of accurate state root confirmation, highlighting the need for further advancements. The inability of light clients to handle zero-knowledge proofs compounds the complexity. Moreover, portal nodes, a crucial bridge in this ecosystem, face the challenge of effectively prioritizing essential information for streamlined and swift stateless light client operations. Overcoming these obstacles is central to empowering light clients and optimizing the Ethereum network for a seamless and inclusive decentralized future.
```

Comment:
Links to portal network specs and maybe specific issue would be great in this section